modules/nf-core/modules/samtools/view/main.nf

```diff
@@ -0,0 +1,37 @@
+// Import generic module functions
```

Comment:
Some of the samtools modules (including this one) have recently been updated. Might be worth to run the modules update command before release